File: backend/src/main/java/com/debbech/devwall/model/ai/Post.java
package com.debbech.devwall.model.ai;


import jakarta.persistence.*;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Entity
@Getter
@Setter
@ToString
public class Post {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String title;
    private String createdAt;
    private String body;

    @OneToOne
    private WriteRequest writeRequest;
    @OneToOne
    private WriteResponse writeResponse;

}

File: dash/src/app/model/Post.ts
import { PostTag } from "./PostTag";

export class Post{
    title : string = "";
    createdAt : string = "";
    id : number = 0;
    body : string = "";
    useful : number = 0;
    notUseful : number = 0;
    tags : PostTag[] = []

    constructor(title : string, createdAt : string, id : number, body : string){
        this.title = title
        this.createdAt = createdAt
        this.id = id
        this.body = body
    }
}